libs/langchain/ragstack_langchain/graph_store/base.py

```diff
@@ -245,14 +248,16 @@ async def atraversal_search(
             Retrieved documents.
         """
         iterator = iter(
-            await run_in_executor(None, self.traversal_search, query, k=k, depth=depth, **kwargs)
+            await run_in_executor(
+                None, self.traversal_search, query, k=k, depth=depth, **kwargs
+            )
         )
         done = object()
         while True:
             doc = await run_in_executor(None, next, iterator, done)
             if doc is done:
                 break
-            yield doc
+            yield doc  # type: ignore[misc]
```

Comment:
Curious what the lint is complaining about? What are we suppressing here?

Reply:
I'm aligning with the langchain code so it's easier to port changes.
This is to appease mypy.
Basically because run_in_executor messes the types.